packages/graphql-server-fastify/package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
It doesn't seem necessary to introduce a `graphql-server-fastify` package.  These are only kept around because they did exist at one point, but they are now deprecated.  Let's just not include these and have `apollo-server-fastify` be the sole package.

Reply:
Sure. I will remove it